source/PlayerInfo.cpp

```diff
@@ -244,7 +244,12 @@ void PlayerInfo::Load(const string &path)
 					reputationChanges.emplace_back(
 						GameData::Governments().Get(grand.Token(0)), grand.Value(1));
 		}
-
+		else if(child.Token(0) == "tribute received")
+		{
+			for(const DataNode &grand : child)
+				if(grand.Size() >= 2)
+					tributeReceived[GameData::Planets().Get(grand.Token(0))] = grand.Value(1);
```

Comment:
tributes are tied to the player and not account because they will be used to see if you can land too, right?

Reply:
I was seriously doubting if I should put tribute under PlayerInfo or under Account. And either is fine for me actually.

Reply:
honestly, since account is contained within playerinfo which is already a big class may as well move to account?

Reply:
I thought about this more and came to the conclusion that tribute should be together with "visited" in some kind of player worldview class; receiving tribute is a result of dominating some planet, and I feel that Account shouldn't have dependencies on Politics or Planets.

So if you don't have a very strong preference, then I like to keep this as-is for this PR. (And have this worldview class for some follow-up PR.)

Reply:
sounds good (we could also just keep it in playerinfo its not that big of a deal)